import-dashboards.py

```diff
@@ -23,7 +23,7 @@
 
 import requests
 
-GRAFANA_DB_DIR             = sys.argv[1] if len(sys.argv) > 1 else '/srv/grafana'
+GRAFANA_DB_DIR             = sys.argv[1] if (len(sys.argv) > 1 and sys.argv[1] != '-d') else '/srv/grafana'
```

Comment:
Maybe it is time to use [argparse](https://docs.python.org/2.7/library/argparse.html)

Reply:
Done